packages/flutter/lib/src/widgets/tap_region.dart

```diff
@@ -635,5 +635,6 @@ class TextFieldTapRegion extends TapRegion {
     super.onTapInside,
     super.consumeOutsideTaps,
     super.debugLabel,
-  }) : super(groupId: EditableText);
+    super.groupId = EditableText,
```

Comment:
Adding `groupId` to `TextFieldTapRegion` feels a little weird because it basically turns it into `TapRegion`, but without as many options. This is probably OK, given it is meant for a specific use case, but technically odd.

Reply:
Yes, thank you for your reminder